Cvičení 1/LightSystem.cpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <glm/glm.hpp>
+#include "LightSystem.h"
+#include "Shader.h"
+#include "SimpleLight.h"
+
+void LightSystem::add(const SimpleLight& light)
+{
+    //podle n�jak�ho p�rka na StackOverflow je "emplace_back" lep�� ne� "push_back" ale kdo v�. Kr�sy 3. str�nky po google search
+	lights.push_back(light);
+}
+
+void LightSystem::calc(Shader& meshShader, Shader& terrainShader)
```

Comment:
Okay, co se stane, když se rozhodnu přidat třetí shader? Jsi na správné cestě, ale trochu to budu muset dotáhnout :)